src/Main.elm

```diff
@@ -59,6 +73,13 @@ init =
                         Date.compare (watchDate m1) (watchDate m2)
                     )
         , selectedMovie = NotSelected
+        , genres = genres
+        , selectedGenres = Set.empty
+        , genresMultiselect =
+            genres
+                |> Set.map (\m -> ( m, toUpper m ))
```

Comment:
Takes each item in a `Set<string>` and turns it into a tuple of `("thestring", "Thestring")`